client/js/nonprofits/supporters/index/merge_supporters.js

```diff
@@ -73,7 +73,12 @@ appl.def('merge.submit', function(form_object, node){
         appl.notify('Error: ' + formatErr(resp))
       }
 		})
+	resetForm()
 	appl.close_modal()
 	appl.wizard.reset('merge_wiz')
 })
 
+const resetForm = function() {
+	document.querySelectorAll('[id^="merge_"]').forEach(function(el) { el.value = '' });
```

Comment:
I thought about adding this to also reset the data from `merge.data.new_supporter.<merge_key>` everytime a merge is done, my guess is that the problem can be there, but I couldn't recreate the case for that to see if it fixes the problem.

```suggestion
	document.querySelectorAll('[id^="merge_"]').forEach(function(el) { el.value = '' });
		appl
		.def('merge.data.new_supporter.name', '')
		.def('merge.data.new_supporter.email', '')
		.def('merge.data.new_supporter.phone', '')
		.def('merge.data.new_supporter.address', '')
		.def('merge.data.new_supporter.city', '')
		.def('merge.data.new_supporter.state_code', '')
		.def('merge.data.new_supporter.zip_code', '')
		.def('merge.data.new_supporter.country', '')
```

Reply:
Have you tested that this works :)

Reply:
So, because I couldn't recreate the problem we have with the wrong data on the new supporter, I couldn't test this, that's why I don't know if it fixes the problem, but I left the suggestion here in case we find the problem in the future.

Reply:
I more mean, does it work in a way that doesn't break anything else?

Reply:
I tested and it didn't break anything.